ahoy.go

```diff
@@ -100,38 +100,39 @@ func getConfig(sourcefile string) (Config, error) {
 
 func getSubCommands(path string) []cli.Command {
 	subCommands := []cli.Command{}
-	if path != "" {
-		includes := strings.Split(path, "\n")
-		commands := map[string]cli.Command{}
-		for _, include := range includes {
-			if len(include) == 0 {
-				continue
-			}
-			includeSource := include
-			if include[0] != "/"[0] || include[0] != "~"[0] {
-				includeSource = filepath.Join(sourcedir, include)
-			}
-			if _, err := os.Stat(includeSource); err != nil {
-				//Skipping files that cannot be loaded allows us to separate
-				//subcommands into public and private.
-				continue
-			}
-			config, _ := getConfig(includeSource)
-			includeCommands := getCommands(config)
-			for _, command := range includeCommands {
-				commands[command.Name] = command
-			}
+	if path == "" {
+		return subCommands
+	}
+	includes := strings.Split(path, "\n")
+	commands := map[string]cli.Command{}
+	for _, include := range includes {
+		if len(include) == 0 {
+			continue
 		}
-
-		var names []string
-		for k := range commands {
-			names = append(names, k)
+		includeSource := include
+		if include[0] != "/"[0] || include[0] != "~"[0] {
+			includeSource = filepath.Join(sourcedir, include)
+		}
+		if _, err := os.Stat(includeSource); err != nil {
+			//Skipping files that cannot be loaded allows us to separate
```

Comment:
I'm not a fan of failing silently here, though I get the use-case.. What if we prefix lines that should not cause the parser to fail with a special character? I'm thinking something that isn't allowed by filesystems and would convey a clear message to not crash if a specific file isn't found. How about '?' .. so it would be like `?~/this-file-might-not-exist.yml` ?